packages/streamsets/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/thomasganka/helm-charts/blob/master/docs/stable/control-agent-2.0.1.tgz?raw=true
+url: https://github.com/thomasganka/helm-charts/raw/master/docs/stable/control-agent-2.0.1.tgz
```

Comment:
@PennyScissors can you double check with the partner that this looks right?

Reply:
confirmed, this one is good.